### DISCUSSION TITLE
to create function in bang bang

### DISCUSSION THREAD

**Discussion by shangamithirai** [shangamithirai/bangbang-covid-19-tracker#3] [Ideas]
can some one suggest some points for function?

**Comment by btwimkomal** [comment]
lets make corner in a website where it can track covid positive around us

**Comment by btwimkomal** [comment]
> can some one suggest some points for function?
lets make corner in a website where it can track covid positive around us

**Reply by shangamithirai** [reply]
nice idea komal

**Reply by Kayal-vizhi0430** [reply]
> > can some one suggest some points for function?
> > lets make corner in a website where it can track covid positive around us

let it alternatively update after before 10 to 15 min

**Reply by shangamithirai** [reply]
are you done with objectives?

**Reply by btwimkomal** [reply]
> are you done with objectives?

wowww yea yea good idea

**Reply by Kayal-vizhi0430** [reply]
thankyou

**Reply by btwimkomal** [reply]
> are you done with objectives?

yea i have made the plan in mind..let me just explain it to you

**Reply by shangamithirai** [reply]
good idea kayal

**Reply by shangamithirai** [reply]
sure!!
